subprojects/docs/src/docs/release/notes.md

```diff
@@ -54,6 +54,35 @@ The allowed labels are:
 
 More details can be found in the [Gradle Wrapper](userguide/gradle_wrapper.html#sec:adding_wrapper) section.
 
+### Kotlin DSL
+
+#### Experimental Property assignment for Kotlin scripts
```

Comment:
:x: I think we should either say `Property` (as the type, with a monotype font) or just "property" (lowercase, normal font).

Reply:
Sounds good.